project/NodeJSEnvForcePolyfills.scala

```diff
@@ -46,6 +46,8 @@ final class NodeJSEnvForcePolyfills(config: NodeJSEnv.Config) extends JSEnv {
           |
           |delete global.Promise;
           |delete global.Symbol;
+          |delete global.Map;
```

Comment:
Ach. Unfortunately it seems that this breaks the `source-map-support` npm package, which we rely on for source map tests. I had not anticipated that.

I think the only solution is to rollback the changes here, and instead of assuming that `Map` and `Set` are present, to assume that `scala.scalajs.LinkingInfo.assumeES6` is true instead.

Reply:
Fixed so.